src/modalities/__main__.py

```diff
@@ -205,6 +206,10 @@ def construct_components(
         wrapped_model, optimizer = self.get_model_and_optimizer(
             config=config, running_env=running_env, checkpointing=checkpointing
         )
+        if config.training.do_apply_activation_checkpointing:
+            apply_activation_checkpointing_inplace(wrapped_model)
+            logging.info("Applied activation checkpointing!")
```

Comment:
Do we need this logging information? If so, I would say "enabled", rather than "applied".